### PR TITLE
changelog for beta

### PR DESCRIPTION
**Pull request by saumyaj3**
# Description
Changelog 5.7 beta

CURA-11714

### CODE REVIEW

File: resources/texts/change_log.txt [hunk 1]
```diff
@@ -1,3 +1,72 @@
+[5.7]
+* New features and improvements:
```
Comment (wawanbreton) [inline resources/texts/change_log.txt:2]: Can you add a blank line here to keep consistency with other versions

**Review by wawanbreton — COMMENTED** [review]
One very minor remark...

File: resources/texts/change_log.txt (hunk 1 above)
Comment (saumyaj3) [reply resources/texts/change_log.txt:3]: [d42f894](https://github.com/Ultimaker/Cura/pull/18615/commits/d42f894965fba80f1f4fad47b957e09a071509e2)

File: resources/texts/change_log.txt [hunk 2]
```diff
@@ -1,3 +1,70 @@
...
+- New Primetower option; Sparse, next to the current Bucket, contributed by @wawanbreton
+- Brim only on the inside setting
+- Allowing to define print sequence manually, contributed by @alexandr-vladimirov (Github)
+- Introduced a setting to apply Monotonic ordering to the Raft Top Surface
+- Introduced a Raft Wall Line Count for the Bottom, Middle and Top of the raft
+- The play speed of the nozzle in the preview is now closer to the movement of the printhead
+- Introduced an option to choose Drop Model to Buildplate every time a project file is loaded
+- Introduced DisplayInfoOnLCD Post Processing Script which combines DisplayFilenameAndLayerOnLCD with DisplayProgress, contributed by @GregValiant
+- Introduced alternative camera navigation similar to FreeCAD Trackpad, contributed by @quarnster
+- Introduced a way to set print sequence manually when printing One at a time, contributed by @alexandr-vladimirov
```
Comment (fieldOfView) [inline resources/texts/change_log.txt:17]: Duplicate of 'Allowing to define print sequence manually' in line 10

Comment (saumyaj3) [reply resources/texts/change_log.txt]: [b8e4251](https://github.com/Ultimaker/Cura/pull/18615/commits/b8e4251e6a2593aaeeb40f7ac536b17d36e86ccf)

Comment (wawanbreton) [reply resources/texts/change_log.txt]: Thanks @fieldOfView for pointing this !

**Pull request merged**